### PR TITLE
Reduce the amount of logging output for webpack

### PR DESCRIPTION
**Pull request by alexose**
This makes it a little more pleasant to develop under `npm run watch` mode-- I was getting so much output that it would occasionally make my terminal lag.

### CODE REVIEW

File: script/build.js [hunk 1]
```diff
@@ -205,6 +214,6 @@ if (options.watch) {
 }
 
 smith.build(function(err) {
-    if (err) throw err;
-    console.log('Build finished!');
-  });
+  if (err) throw err;
+  console.log('Metalsmith build finished!  Now starting webpack...');
```
Comment (awong-dev) [inline script/build.js]: Is this actually the right ordering? I thought webpack would have finished before this is executed...

**Comment by awong-dev** [comment]
LGTM with one comment.

File: script/build.js (hunk 1 above)
Comment (alexose) [inline script/build.js]: This would be better written as 'Now starting webpack-dev-server' for watch mode only.  I'll make that change...

Comment (alexose) [inline script/build.js]: Actually, the behavior is slightly different for `--watch`.  Webpack seems to run afterwards.

**Pull request merged**